test/run_tests.sh

```diff
@@ -22,7 +22,7 @@ echo 'artifactsDir: /tmp/kudo-e2e-test' >> kudo-e2e-test.yaml.tmpl
 
 # Pull the builder image with retries if it doesn't already exist.
 retries=0
-builder_image=$(awk '/FROM/ {print $2}' test/Dockerfile)
+builder_image=$(awk '/FROM/ {print $2}' test/Dockerfile.amd64)
```

Comment:
This is `test/Dockerfile`, which wasn't renamed. Revert the changes in this file to make the integration and e2e tests succeed.

Reply:
```suggestion
builder_image=$(awk '/FROM/ {print $2}' test/Dockerfile)
```